src/GeneralStateTestsFiller/stTransactionTest/CreateTransactionSuccessFiller.json

```diff
@@ -9,18 +9,17 @@
             "previousHash" : "5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6"
         },
         "expect" : [
-    	    {
+            {
                 "indexes" : {
                     "data" : -1,
                     "gas" : -1,
                     "value" : -1
                 },
-                "network" : [">=Istanbul"],
+                "network" : ["Frontier","Homestead",">=Istanbul"],
```

Comment:
we have this test for Frontier and Homestead in legacy, why ?

Reply:
I didn't know they existed. Anyway, they have different format so I'm not able to use these.